ca.parallel_launcher.ParallelLauncher.json

```diff
@@ -0,0 +1,52 @@
+{
+	"id": "ca.parallel_launcher.ParallelLauncher",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.15-21.08",
+	"sdk": "org.kde.Sdk",
+	"command": "parallel-launcher",
+	"modules": [
+		{
+			"name": "parallel-launcher",
+			"buildsystem": "qmake",
+			"config-opts": [ "CONFIG+=flatpak-version" ],
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://gitlab.com/parallel-launcher/parallel-launcher.git",
+					"branch": "a4c436b0c8077fdbb2ee4aeb68b15d95e7bfb5db"
+				},
+				{
+					"type": "file",
+					"path": "ca.parallel_launcher.ParallelLauncher.metainfo.xml"
+				}
+			],
+			"post-install": [
+				"install -D -t /app/share/parallel-launcher/translations/ lang/*.qm",
+				"mkdir -p /app/share/icons/hicolor/128x128/apps/",
+				"mv -f /app/share/icons/appicon.svg /app/share/icons/hicolor/128x128/apps/ca.parallel_launcher.ParallelLauncher.svg",
+				"mv -f /app/share/mime/packages/bps-mime.xml /app/share/mime/packages/ca.parallel_launcher.ParallelLauncher.xml",
+				"install -D -t /app/share/metainfo/ ca.parallel_launcher.ParallelLauncher.metainfo.xml"
+			]
+		},
+		"retroarch/retroarch.json",
+		"retroarch/retroarch-filters-video.json",
+		"retroarch/retroarch-filters-audio.json",
+		"retroarch/retroarch-assets.json",
+		"retroarch/libretro-database.json",
+		"retroarch/libretro-core-info.json"
+	],
+	"finish-args" : [
+		"--socket=x11",
```

Comment:
All permissions here are the same as the ones required by [RetroArch](https://github.com/flathub/org.libretro.RetroArch) (which this application launches). One exception is that I removed the `--filesystem=xdg-run/app/com.discordapp.Discord:create` permission, because I don't need RetroArch to have that for my use case.

Reply:
Nevermind, added this permission back since I do actually need it